syntax = "proto3";

message Proto_CreatePriChannel {
  repeated string user_name = 1;
  repeated bytes channel_name = 2;
  repeated bytes client_type = 3;
  repeated bytes client_id = 4;
}

